waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/security/DatabaseWhitelistAccessControlHandler.java

```diff
@@ -16,21 +16,23 @@
 package com.hotels.bdp.waggledance.server.security;
 
 import java.util.ArrayList;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import com.hotels.bdp.waggledance.api.federation.service.FederationService;
 import com.hotels.bdp.waggledance.api.model.PrimaryMetaStore;
 
 public class DatabaseWhitelistAccessControlHandler implements AccessControlHandler {
 
-  private final Set<String> databaseWhiteList = new HashSet<>();
+  private final List<Pattern> databaseWhiteList = new ArrayList<>();
```

Comment:
Why the change from Set to List? Duplicates don't really make sense here so I think a Set is better.